planner/core/explain.go

```diff
@@ -246,20 +246,6 @@ func (p *PhysicalTableScan) AccessObject(normalized bool) string {
 // OperatorInfo implements dataAccesser interface.
 func (p *PhysicalTableScan) OperatorInfo(normalized bool) string {
 	var buffer strings.Builder
-	for i, pkCol := range p.PkCols {
-		switch i {
-		case 0:
-			buffer.WriteString("pk cols: (")
```

Comment:
Why do you want to remove this information？

Reply:
Because the `p.PkCols` is never initialized.